client/packages/common/src/ui/components/inputs/TextInput/TextInput.stories.tsx

```diff
@@ -66,22 +63,32 @@ const NumericTemplate: Story = () => {
           <Grid item xs>
             <StyledPaper>
               <Typography>NumericTextInput</Typography>
-              <NumericTextInput />
+              <NumericTextInput
```

Comment:
This one could have a more accurate title now. (2 decimal places, allows negatives)
It would be good to add an example showing the defaults. You could have
* default options
* allows negative
* max value set

Reply:
Done 👍🏼